style.css

```diff
@@ -0,0 +1,57 @@
+
+body {
+    padding: 0;
+    margin: 0;
+    background: #160808;
+}
+
+div {
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    width: 150px;
+    height: 180px;
+    border: 6px solid #262626;
+    border-top: 2px solid transparent;
+    border-radius: 15px;
+    border-top-right-radius: 5px;
+    border-top-left-radius: 5px;
+    background: url(image1.png);
+    background-size: 450% 100%;
+    background-repeat: repeat-x;
+    box-shadow: 0 0 0 6px white;
```

Comment:
why aren't we grouping css properties ?

Reply:
u mean, shorthand?

Reply:
pls, check now

Reply:
not shorthand grouping means packing similar properties together and separating them for better readability and understanding of the document example shown in image
![image](https://user-images.githubusercontent.com/91013960/198897835-04eb3e43-92d5-41de-a51c-84cbf00c6a87.png)